frontend/tests/e2e/features/addStudent.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Register a new student
+    As a user
+    I want to register new students
+
+    Background:
+        Given the user has browsed the home page
+
+    Scenario: Successful student registration with no ielts score
```

Comment:
```suggestion
    Scenario: student registration without IELTS score
```
`Then` step will signify whether the scenario is supposed to be successful. I think this will be a better scenario title